mlir/test/Target/LLVMIR/omptarget-array-sectioning-host.mlir

```diff
@@ -0,0 +1,80 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
```

Comment:
Could you minimize this test? May be run the MLIR CSE, DCE passes on it?

Reply:
I can certainly try!